### PR TITLE
Connect to CoreUpdate

### PR DESCRIPTION
**Pull request by efinlay24**
- connect our k8s clusters to the internal coreupdate server

### CODE REVIEW

File: userdata/cloud-config-controller [hunk 1]
```diff
@@ -20,6 +20,8 @@ exec /usr/bin/coreos-cloudinit --from-file /var/run/coreos/$USERDATA_FILE
 coreos:
   update:
     reboot-strategy: "off"
+    group: upp-k8s-eu-west-1-prod-controller
```
Comment (sbuliarca) [inline userdata/cloud-config-controller]: Do we need a different group for pac or other clusters? As this cloud-init is universal

**Review by mclarke47 — APPROVED** [review]

File: userdata/cloud-config-controller (hunk 1 above)
Comment (efinlay24) [reply userdata/cloud-config-controller]: Hmm, good point.

Do we have separate userdata files for those?

Comment (sbuliarca) [reply userdata/cloud-config-controller]: Nope, we've been trying to use the same one, that is generated by kube-aws, and not touch it afterwards, as on kube-aws upgrades this file may change.

Comment (efinlay24) [reply userdata/cloud-config-controller]: Ok, in that case we should be able to use some kind of env variable that we can reference in the template to generate the group name.

Do we already have a `upp` / `pac` env variable available?

File: userdata/cloud-config-controller [hunk 2]
```diff
@@ -20,6 +20,8 @@ exec /usr/bin/coreos-cloudinit --from-file /var/run/coreos/$USERDATA_FILE
 coreos:
   update:
     reboot-strategy: "off"
+    group: {{.ClusterName}}-controller
```
Comment (efinlay24) [inline userdata/cloud-config-controller:23]: @mclarke47 @sbuliarca:

Ok, so this should expand out to:

```
pre-prod-eu-pac-controller
prod-eu-upp-controller
```
etc, etc.

Thoughts? Does that work for everyone?

Comment (efinlay24) [reply userdata/cloud-config-controller:23]: (for Matt's reference - UPP and PAC are being considered to be two separate platforms, hence the extra confusion)

Comment (sbuliarca) [reply userdata/cloud-config-controller:23]: Indeed I think this is the best option so far. We'll need to remember to include these lines in future kube-aws upgrades, but this should be easily spotted on PRs

**Review by sbuliarca — APPROVED** [review]

**Comment by mclarke47** [comment]
Ok so all that needs to be done now is:
1. Add your groups to this file: https://github.com/Financial-Times/ansible-coreos-update/blob/master/group_vars/euprod/coreos#L18 (so that if I need to recreate the server they will get created)
2. Create your groups manually through the UI:
https://coreupdate-eu-prod.memb.ft.com/cp/apps

(unfortunately there's no updating group through ansible because of the coreupdate api nonsense I had to do)

**Pull request merged**